int8 da_id 
string type
string state_name
string state_input
float32 priority
float32 deadline
# task state, 0 = initialised, 1 = commanding, 2 = suspending, 3 = suspended, 4 = resuming, 5 = resuming
int8 state